frontend/src/libs/components/backward-button/backward-button.tsx

```diff
@@ -1,15 +1,23 @@
 import { IconColor } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
 
 import { Icon } from '../components.js';
 import styles from './styles.module.scss';
 
 type Properties = {
   onGoBack: () => void;
+  isVisible?: boolean;
```

Comment:
it is useless

Reply:
Well, I rewrote the styles of the back button, and as we need this arrow on the big screen only for the meditation player, so I need this

Reply:
But we need to handle the visibility outside the button

Reply:
Well, I need to change the left position for the button if it is a desktop, and if I return back the style of the back button where it has display - none in desktop resolution, how could I handle it in meditation player component if I want to have it visible?